frontend/src/feature-flags/excludeFeatureRules.ts

```diff
@@ -0,0 +1,12 @@
+import {ExcludeFeatureRule} from './featureFlagsProvider'
+
+export const EXCLUDE_FEATURE_RULES: ExcludeFeatureRule[] = [
+  (feature, minorVersion) => feature === 'fsx_ontap' && minorVersion < 2,
```

Comment:
issue: this should be `<= 2`, as well as the other features, since it has been introduced with PC 3.2.0

Reply:
Currently we are checking that `versionMinor >= 2` to allow this feature, therefore in the exclude rule I believe it's correct to check that it's `< 2`

Reply:
Yes, you're right. However, it could be better to think on "allowing" rather than "excluding" as it probably makes more sense in the feature flags context.

Reply:
As per our discussion, I refactored into a mechanism based on "allowing" rather than "excluding"